src/app/(proper_react)/(redesign)/Footer.tsx

```diff
@@ -30,6 +39,13 @@ export const Footer = ({ l10n }: { l10n: ExtendedReactLocalization }) => {
             {l10n.getString("footer-nav-all-breaches")}
           </Link>
         </li>
+        {countryCode === "us" && !session && (
+          <li>
+            <a href="/how-it-works" target="_blank">
```

Comment:
Why open in a new tab if this is same-domain?

Reply:
Leaving unchanged per PM request

Reply:
I think it would be still good to take advantage of `<Link>` from Next.js for internal links. It uses App Router and takes care of things like pre-fetching. We can still pass `target="_blank"` in order to open it in a new tab/window.